### DISCUSSION TITLE
Bootstrapt sass import issue

### DISCUSSION THREAD

**Discussion by medeepaksaini** [twbs/bootstrap#31629] [General] [answered]
hello, i'm trying to import bootstrap function using `@import '~bootstrap/scss/functions';` but getting error 
`@import '~bootstrap/scss/functions';`

Can you please tell me why we use `~`

**Comment by Deckluhm** [comment, marked as answer]
`~` is a webpack shortcut to `node_modules`: https://webpack.js.org/loaders/css-loader/#import
